c/src/CMakeLists.txt

```diff
@@ -31,10 +31,27 @@ set(libsbp_SRCS
   edc.c
   sbp.c
   )
-set(LIBSBP_CFLAGS "-fpic" CACHE STRING "Compile flags for libsbp.")
+
+set(LIBSBP_CFLAGS "-fpic -Wall -Werror" CACHE STRING "Compile flags for libsbp.")
```

Comment:
Please check this conversation

https://github.com/swift-nav/libsbp/pull/647#discussion_r256539614
and
https://github.com/swift-nav/libsbp/pull/647#discussion_r256899431

Reply:
Amazing! Thanks for the info.